src/definition/scheduler/Schedule.ts

```diff
@@ -0,0 +1,36 @@
+import { Recurrence, ScheduleData } from 'later';
+import { parse } from 'later';
```

Comment:
What about i18n for later? I've only seen english. If this is the only option, a reusable GUI component is elementary

Reply:
You bring up a very very good point, one which I need to reevaluate. I created this pull request in order to get feedback, so thank you for bringing this to my attention! :)

Reply:
As of right now, we will be switching to crontab style only. Support for i18n will come in the future.